docs/using-az-aro.md

```diff
@@ -35,6 +35,15 @@ against a development RP running at https://localhost:8443/.
    Note: you will be able to update the `az aro` extension in the future by
    simply running `git pull`.
 
+1. Prepare a Python development environment:
+
+   ```
+   PYTHON_VERSION=3 make pyenv3  # Can also specify Python 2 if necessary
+   source pyenv3/bin/activate
```

Comment:
fwiw `make pyenv; . pyenv/bin/activate` wfm and uses the system default python version

Reply:
That doesn't WFM because I don't have a `PYTHON_VERSION` exported by default, so it fails... does that come automatically in a Fedora env?

Reply:
```
$ ls -l /bin/python
lrwxrwxrwx 1 root root 9 Jan 30 03:18 /bin/python -> ./python3
```